### DISCUSSION TITLE
Make commit button sticky

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#38998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When making changes to a large file and the changes are at the lower parts of it is it always tedious to scroll back up to press the "Commit changes..." button to get your changes commited.

It would be a great improvement to have the button "sticky", staying at the top of the screen while scrolling down, similar to how the bar with the various buttons (Raw view, edit, etc) is sticking with you when scrolling down.

**Comment by jfullstackdev** [comment]
is it the new Code Search or GitHub Web UI ? 

because in GitHub Desktop, the commit is there bottom left no matter how big is the file or the changes

![image](https://user-images.githubusercontent.com/47092464/201528241-56a7b402-7493-4902-adb0-a233fd49dd0e.png)

**Reply by Andre601** [reply]
I wouldn't talk about Desktop here... There is a more suitable place for that and given that this has the label "Code Search and Navigation", it should be clear that it is about the Web UI.

**Reply by jfullstackdev** [reply]
yup, that's why I'm asking

as for now, as you can see GitHub is triaging a lot of issues and feature requests, this might not be in their roadmap, even their Support cannot respond to the multitude of request

so that's why I'm giving that option, it's just easy to install and use GitHub Desktop
